src/ontology/templates/biobank-specimens.tsv

```diff
@@ -1,72 +1,79 @@
+OBI:0002554	vagina specimen		A specimen that is derived from vagina.	"Chris Stoeckert, NCI BBRB "	equivalent	vagina		UBERON_0000996	Chris Stoeckert	IAO:0000122
+OBI:0002603	areolar swab specimen		A specimen that is collected with a swab from the surface of an areola of a breast.	"Chris Stoeckert, OBIB "	equivalent		areola	UBERON_0002032	Chris Stoeckert	IAO:0000122
+OBI:0002604	breast swab specimen		A specimen that is collected with a swab from the surface of a breast.	"Chris Stoeckert, OBIB "	equivalent		breast	UBERON_0000310	Chris Stoeckert	IAO:0000122
+OBI:0002605	cheek swab specimen		A specimen that is collected with a swab from the cheek portion of the inside surface of a mouth.	"Chris Stoeckert, OBIB "	equivalent		cheek	UBERON_0001567	Chris Stoeckert	IAO:0000122
+OBI:0002606	nasopharyngeal swab specimen		A specimen that is collected with a swab from the surface of a nasopharynx.	"Chris Stoeckert, OBIB "	equivalent		nasopharynx	UBERON_0001728	Chris Stoeckert	IAO:0000122
+OBI:0002607	oral swab specimen		A specimen that is collected with a swab from the inside surface of a mouth.	"Chris Stoeckert, OBIB "	equivalent		mouth	UBERON_0000165	Chris Stoeckert	IAO:0000122
+OBI:0002608	oropharyngeal swab specimen		A specimen that is collected with a swab from the surface of a oropharynx.	"Chris Stoeckert, OBIB "	equivalent		oropharynx	UBERON_0001729	Chris Stoeckert	IAO:0000122
+OBI:0002609	rectal swab specimen		A specimen that is collected with a swab from the surface of a rectum.	"Chris Stoeckert, OBIB "	equivalent		rectum	UBERON_0001052	Chris Stoeckert	IAO:0000122
+OBI:0002610	tongue swab specimen		A specimen that is collected with a swab from the surface of a tongue.	"Chris Stoeckert, OBIB "	equivalent		tongue	UBERON_0001723	Chris Stoeckert	IAO:0000122
+OBI:0002611	vagina swab specimen		A specimen that is collected with a swab from the surface of a vagina.	"Chris Stoeckert, OBIB "	equivalent		vagina	UBERON_0000996	Chris Stoeckert	IAO:0000122
+OBI:0002612	fecal swab specimen	stool swab specimen	A specimen that is collected with a swab from the surface of a feces.	"Chris Stoeckert, OBIB "	equivalent		feces	UBERON_0001988	Chris Stoeckert	IAO:0000122
+OBI:0002613	environmental swab specimen		A specimen that is collected with a swab from the surface of an environmenal material.	"Chris Stoeckert, OBIB "	equivalent		environmental material	ENVO_00010483	Chris Stoeckert	IAO:0000122
 OBI:0002746	induced sputum specimen		A sputum specimen that is collected from an organism following its inhalation of a nebulized salt solution.	PMC3297553	subclass	sputum		UBERON_0007311	John Judkins	
+OBI:0002778	tracheal aspirate specimen	tracheal wash specimen	a specimen that is collected from the tracheal anatomic part (the tube connecting larynx to primary bronchi) of the respiratory system of an organism using a suction motion	"combined from wiki, pubmed, and dictionary"	subclass		trachea	UBERON_0003126	Asiyah Yu Lin|ORCID:0000-0003-2620-0345	
```

Comment:
I know the term requester submitted these definitions & sources so you can replace the source if you rewrite the defintion. It looks like she didn't provide any specifics. Here are just some notes on improving this entry:

This definition can be simplified to match other definitions in this sheet (I didn't do a ton of research into this, so this might not be fully accurate):
> A specimen that is collected with suction from the trachea.

For definition sources, please use a pipe separator (`|`) instead of comma-separation. There should be no spaces between entries. See examples starting at line 77.

Also for definition sources, please provide specifics. If you are using an article from PubMed, include the ID (e.g., `PMID:1234`). For Wikipedia, just include the whole link. If the dictionary was an online dictionary, also include the link to the page that you got the detail from. The whole defintion source may end up looking like:
```
PMID:1234|http://en.wikipedia.org/wiki/Article_name|http://www.dictionary.com/term
```